jenkins-scripts/lib/check_graphic_card.bash

```diff
@@ -80,6 +83,8 @@ if $GPU_SUPPORT_NEEDED; then
       echo "GPU support needed by the script but DISPLAY var is empty"
       # Try to restart lightdm. It should stop the script in the case of failure
       sudo service lightdm restart
+      # Wait for lightdm service to restart X11.
+      sleep 5
```

Comment:
5 seconds is possibly way too long, but if this does not work as expected we likely need a different hypothesis.

Reply:
Yeah I'm not certain that the watching lightdm activation / status would be sufficient to resolve the race since I don't know if lightdm uses systemd's notification API for actually declaring readiness or if it's a simple/exec type service which is considered active as soon as the process runs https://www.freedesktop.org/software/systemd/man/systemd.service.html in which case we'd still need to wait ourselves for X11 to start.